<launch>
    <rosparam command="load" file="$(env USER_HOME)/rubis_ws/src/rubis_autorunner/cfg/cubetown_autorunner/cubetown_autorunner.yaml" />
    <node pkg="rubis_autorunner" type="cubetown_autorunner" name="cubetown_autorunner" output="screen" />
</launch>